feat(helm:create): allow absolute paths when specifying starter template path [master]

mentos1386 opened the pull request:
**What this PR does / why we need it**:

If starter template path is an absolute path, it shouldn't be prefixed with c.home.Starters() but rather be used as is. This way we can use templates that might not be in home directory but inside a specific project.

PR to `dev-v3` branch is #5985  
**Special notes for your reviewer**:

**If applicable**:
- [x] this PR contains documentation
- [x] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

bacongobbler commented:
thanks for the PR! Would you be willing to port this over to the `dev-v3` branch as well? That way it'll make its way into Helm 3. Thanks!

bacongobbler commented:
Also would you mind writing a unit test for this?

mentos1386 commented:
@bacongobbler should i close this PR and create a new one for `dev-v3`? Or create another one to `dev-v3`?

bacongobbler commented:
Create a similar one for dev-v3

mentos1386 commented:
@bacongobbler Hey, just want to inform you that this is done from my side. Also the dev-v3 version

bacongobbler commented on cmd/helm/create.go:
```diff
@@ -106,6 +107,10 @@ func (c *createCmd) run() error {
 		// Create from the starter
 		lstarter := filepath.Join(c.home.Starters(), c.starter)
+		// If path is absolute, we dont want to prefix it with helm starters folder
+		if strings.HasPrefix(c.starter, "/") {
```
Can you use `filepath.IsAbs` instead? https://golang.org/pkg/path/filepath/#IsAbs

mentos1386 replied:
Fixed, also in v3 pr

mentos1386 committed:
feat(helm:create): allow absolute paths

If starter template path is an absolute path, it shouldn't
be prefixed with c.home.Starters() but rather be used as is.

Signed-off-by: Tine Jozelj <tine.jozelj@tjo.space>

mentos1386 commented:
@bacongobbler @mattfarina PR for v3 https://github.com/helm/helm/pull/5985 was merged last week. Shouldn't this be also merged?

bacongobbler commented:
PRs require 2 LGTMs to be merged. So far there's just one. :)

hickeyma reviewed (approved):
LGTM, thanks @mentos1386
